### ISSUE TITLE
AddImportCompletionData null reference exception

### ISSUE THREAD

**Issue by mrward** [mono/monodevelop#6455]
VSMac: 7.7.0.1738
Git revision: 7dc7716c1cf2bdfb1baafd728a09ef805643c15d
Build date: 2018-10-23 14:31:07+00
Build branch: release-7.7
Xamarin extensions: c114276813ea8f259a34bc54b015d7bf711d402b

Do not have a specific repro for this. IDE log had this error a few times. Working with main.sln.

```
Exception while AddImportCompletionData
System.NullReferenceException: Object reference not set to an instance of an object
  at MonoDevelop.CSharp.Completion.CSharpCompletionTextEditorExtension.AddImportCompletionData (Microsoft.CodeAnalysis.CSharp.Extensions.ContextQuery.CSharpSyntaxContext ctx, MonoDevelop.Ide.CodeCompletion.CompletionDataList result, Microsoft.CodeAnalysis.SemanticModel semanticModel, System.Int32 position, System.Threading.CancellationToken cancellationToken) [0x000ca] in /Users/vsts/agent/2.141.1/work/1/s/monodevelop/main/src/addins/CSharpBinding/MonoDevelop.CSharp.Completion/CSharpCompletionTextEditorExtension.cs:288 
```

> VS bug [#714055](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/714055)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-20] Comment by **xamarinc**



Fixed in version 7.7.3.29 (release-7.7)



Pull Request #6783 merged by: Lluis Sanchez  
 Author: mono  
 Commit: e03698fba5d38a77f935b33934e0251d28da56a9 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-22] Comment by **xamarinc**



Fixed in version 8.0.0.1048 (master)



Pull Request #6781 merged by: Lluis Sanchez  
 Author: mono  
 Commit: e08907cd677874935e354f4df837e52f441e6786 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-23] Comment by **xamarinc**



Fixed in version 7.8.0.1509 (release-7.8)



Pull Request #6784 merged by: Lluis Sanchez  
 Author: mono  
 Commit: c4fe168fb146ff7542fa550dbeffc65fa2f75f12 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-24] Comment by **xamarinc**



Fixed in version 8.0.0.1054 (release-8.0)



Pull Request #6781 merged by: Lluis Sanchez  
 Author: mono  
 Commit: e08907cd677874935e354f4df837e52f441e6786 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-29] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Pull Request #6781 merged by: Lluis Sanchez  
 Author: mono  
 Commit: e08907cd677874935e354f4df837e52f441e6786 (xamarin/monodevelop)
